Packs/GCP-IAM/Integrations/GCP-IAM/GCP-IAM.yml

```diff
@@ -0,0 +1,1940 @@
+category: IT Services
+commonfields:
+  id: GCP-IAM
+  version: -1
+configuration:
+- displaypassword: Service Account Private Key file content (JSON).
+  name: credentials
+  required: true
+  type: 9
+  hiddenusername: true
+description: Manage identity and access control for Google Cloud Platform resources.
+display: GCP-IAM
+name: GCP-IAM
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The project name to retrieve. For example, projects/415104041262.
+      isArray: false
+      name: project_name
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: The number of results to retrieve. Minimum value is 1, maximum
+        value is 100.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '1'
+      description: The page number of the results to retrieve. Minimum value is 1.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      description: The name of the parent resource to list projects under. For example,
+        setting this field to 'folders/1234' would list all projects directly under
+        that folder.
+      isArray: false
+      name: parent
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'False'
+      description: Indicate that projects in the DELETE_REQUESTED state should also
+        be retrieved.
+      isArray: false
+      name: show_deleted
+      predefined:
+      - 'False'
+      - 'True'
+      required: false
+      secret: false
+    deprecated: false
+    description: 'List projects under the specified parent, or retrieve project information.
+      One of the arguments: ''parent'' or ''project_name''  must be provided'
+    execution: false
+    name: gcp-iam-projects-get
```

Comment:
@DanilVil please approve the command naming (list+get)

Reply:
@dantavori It's ok, this is the areeg combination of both of the commands